### PR TITLE
Todo functionality & Readme

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR, I'm adding add/delete from todo list functionality.

### CODE REVIEW

File: src/components/Todo.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            />
+            {item.todo}
+          </label>
+        ))}
+      </div>
+      <button
+        className="delete-completed-btn"
+        onClick={() => deleteCheckedTodo()}
+      >
+        Deleted Completed
```
Comment (Ranjeetk489) [inline src/components/Todo.jsx]: Delete Completed*

**Review by Ranjeetk489 — APPROVED** [review]
Nicely structured and readable code and I think i have used extension with similar UI to this.. appreciation for good implementation. good to merge
suggestion- 
disable 'delete completed' button if task list is empty
have scrollbar and show x number of tasks as multiple tasks may cause it too overflow
duplicate tasks shouldn't be allowed

**Comment by sudipkundu999** [comment]
> Nicely structured and readable code and I think i have used extension with similar UI to this.. appreciation for good implementation. good to merge suggestion- disable 'delete completed' button if task list is empty have scrollbar and show x number of tasks as multiple tasks may cause it too overflow duplicate tasks shouldn't be allowed

Thank you for the suggestions. I have implemented all of them. 🚀

File: src/components/component-css/todo.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+.list-display label {
+  cursor: pointer;
+  max-width: 15ch;
+  overflow-wrap: break-word;
+}
+
+.list-display input {
+  --size: 1.75rem;
+  width: var(--size);
+  height: var(--size);
```
Comment (zoyatanweer) [inline src/components/component-css/todo.css:50]: appreciation: liked the way you used consistency and variables here.

**Review by zoyatanweer — APPROVED** [review]
- Loved the UI ! So good looking.
- Great code; easily understandable; nice class naming and use of variables.
- Good to merge.

**Pull request merged**